code/world.dm

```diff
@@ -56,18 +57,32 @@
 			external_rsc_urls.Cut(i,i+1)
 #endif
 
-/world/proc/SetRoundID()
+/world/proc/CheckSchemaVersion()
 	if(config.sql_enabled)
 		if(SSdbcore.Connect())
 			log_world("Database connection established.")
+			var/datum/DBQuery/db_version = SSdbcore.NewQuery("SELECT major, minor FROM[format_table_name("schema_version")]")
```

Comment:
Need a space between `FROM` and `[format_table_name`, otherwise it'd resolve to `FROMSS13_schema_version`